src/main/java/com/toptrumps/core/Player.java

```diff
@@ -1,15 +1,32 @@
 package com.toptrumps.core;
 
+/**
+ * Represents the player (user) of the game.
+ * There should only ever be one player.
+ */
 
 import java.util.ArrayList;
 
 public class Player{
 
+    /**
+     * The name of the player, set by default.
+     */
+    private final String NAME = "Human player"; // Obviously can be changed later
```

Comment:
Why is this variable uppercased?

Reply:
It's a constant. The comment refers to the string which can be changed by us, later, if we want.

Reply:
Okay! It's common practice to declare constants in Java with the "static" modifier:

```private static final String NAME_VALUE = "Human player"```

Assuming that you're using IntelliJ, they have a nice shortcut for it as well: 
https://www.jetbrains.com/help/idea/extract-constant-refactoring-dialog.html